src/view/components/customerIdWrapper.jsx

```diff
@@ -0,0 +1,200 @@
+            render={arrayHelpers => {
+              return (
+                <React.Fragment>
+                  <div className="u-gapTop u-alignRight">
+                    <Button
+                      id="addCustomerId"
+                      label="Add Customer ID"
+                      onClick={() => {
+                        arrayHelpers.push(getDefaultCustomerId());
+                      }}
+                    />
+                  </div>
+                  {((Array.isArray(values.customerIds) &&
+                    values.customerIds.length) ||
+                    null) && (
+                    <Heading variant="subtitle2">Customer IDs</Heading>
+                  )}
+                  <div>
+                    {((Array.isArray(values.customerIds) &&
+                      values.customerIds.length) ||
+                      null) &&
+                      values.customerIds.map((customerId, index) => (
+                        <Well key={index}>
+                          <div>
+                            <FieldLabel
+                              labelFor="namespaceField"
+                              label="Namespace"
+                            />
+                            <div>
+                              <WrappedField
+                                id="namespaceField"
+                                name={`customerIds.${index}.namespace`}
+                                // component={Select}
+                                component={Textfield}
+                                componentClassName="u-fieldLong"
+                                // options={namespaceOptions}
+                              />
+                            </div>
+                          </div>
+                          <div className="u-gapTop">
+                            <FieldLabel labelFor="idField" label="ID" />
+                            <div>
+                              <WrappedField
+                                id="idField"
+                                name={`customerIds.${index}.id`}
+                                component={Textfield}
+                                componentClassName="u-fieldLong"
+                                supportDataElement
+                              />
+                            </div>
+                          </div>
+                          <div className="u-gapTop">
+                            <InfoTipLayout tip="Uses the SHA-256 hashing algorithm that allows you to pass in customer IDs or email addresses, and pass out hashed IDs. This is an optional Javascript method for sending hashed identifiers. You can continue to use your own methods of hashing prior to sending customer IDs.">
+                              <WrappedField
+                                name={`customerIds.${index}.hashEnabled`}
+                                component={Checkbox}
+                                label="Convert ID to sha256 hash"
+                              />
```

Comment:
The tooltip now reads: "Uses the SHA-256 hashing algorithm that allows you to pass in customer IDs or email addresses, and pass out hashed IDs. This is an optional Javascript method for sending hashed identifiers. You can continue to use your own methods of hashing prior to sending customer IDs. Note: if this is set to true for a customer ID and the page is HTTP, the customer ID will be removed from the call because hashing cannot be completed in this case." Waiting on @jileeshs to implement in alloy.

Reply:
The related alloy work will be merged in a future pull request.